aten/src/ATen/core/ivalue_inl.h

```diff
@@ -1136,10 +1137,6 @@ IValue from_(T x, std::true_type) {
 }
 template <typename T>
```

Comment:
let's enable_if this on an `is_base_of` check (or alternatively add a static_assert)

Reply:
See above re static check failing